api/src/main/java/com/nicknathanjustin/streamercontracts/votes/VoteService.java

```diff
@@ -0,0 +1,33 @@
+package com.nicknathanjustin.streamercontracts.votes;
+
+import com.nicknathanjustin.streamercontracts.contracts.ContractModel;
+import com.nicknathanjustin.streamercontracts.users.UserModel;
+
+public interface VoteService {
+
+    /**
+     * Creates a vote given the supplied parameters
+     *
+     * @param voter User that voted on this contract
+     * @param contract Contract user has voted on
+     * @param flaggedCompleted indicates if the user considered the contract completed or not
+     */
+    void recordVote(UserModel voter, ContractModel contract, boolean flaggedCompleted);
+
+    /**
+     * Checks if voting is complete for the given contract
```

Comment:
Can you just elaborate the comment on this a bit? Will this return true even after the contract has been completed as well? Or is it just when the voting is completed, but the contract hasn't been completed yet?

Reply:
"Will this return true even after the contract has been completed as well?"
Yes, its checking if voting is complete as the method header describes. Knowing if voting is complete or not is independent of the contract completing.

Reply:
Can we update the comment to reflect this?

Reply:
I think you meant to comment on getVoteOutcome()... if so that has been updated. If not, im not sure whats being asked here